modules/currency-l1/src/main/scala/org/tessellation/currency/l1/http/EstimatedFeeResponse.scala

```diff
@@ -6,17 +6,18 @@ import org.tessellation.currency.schema.EstimatedFee
 import org.tessellation.currency.schema.EstimatedFee.{Estimated, Zero}
 import org.tessellation.schema.address.Address
 import org.tessellation.schema.balance.Amount
+import org.tessellation.security.hash.Hash
 
 import derevo.circe.magnolia.encoder
 import derevo.derive
 
 @derive(encoder)
-case class EstimatedFeeResponse(fee: Amount, address: Option[Address])
+case class EstimatedFeeResponse(fee: Amount, address: Option[Address], updateHash: Hash)
 
 object EstimatedFeeResponse {
-  def apply(ef: EstimatedFee): EstimatedFeeResponse =
+  def apply(ef: EstimatedFee, updateHash: Hash): EstimatedFeeResponse =
     ef match {
-      case Zero            => EstimatedFeeResponse(fee = Amount.empty, address = none)
-      case Estimated(f, a) => EstimatedFeeResponse(fee = f, address = a.some)
+      case Zero            => EstimatedFeeResponse(fee = Amount.empty, address = none, Hash.empty)
```

Comment:
Why for the no fee the update hash is empty? It should be the `updateHash` as well